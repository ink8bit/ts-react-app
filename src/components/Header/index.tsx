import React from 'react';
import { Link } from '@reach/router';

import s from './styles.css';

const Header: React.FunctionComponent<{}> = () => {
  return (
    <header className={s.header}>
      <Link to="/">Link</Link>
    </header>
  );
};

export default Header;
