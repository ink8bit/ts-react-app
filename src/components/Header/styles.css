.header {
  background: lightcoral;
  padding: var(--offset);
}
