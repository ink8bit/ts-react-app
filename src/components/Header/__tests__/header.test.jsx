import React from 'react';
import { create } from 'react-test-renderer';

import Header from '../index';

test('snapshot', () => {
  const c = create(<Header />);
  expect(c.toJSON()).toMatchSnapshot();
});
