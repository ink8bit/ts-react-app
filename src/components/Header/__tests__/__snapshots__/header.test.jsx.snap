// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<header
  className="header"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    Link
  </a>
</header>
`;
